### PR TITLE
Lunapci hsm setup

### PR DESCRIPTION
**Pull request by om-shrirao**

### CODE REVIEW

File: docs/resources/hsm_root_of_trust_setup.md [hunk 1]
```diff
@@ -55,6 +55,19 @@ resource "ciphertrust_hsm_root_of_trust_setup" "cm_hsm_rot_setup" {
...
+```terraform
+# HSM root of trust setup of type lunapci
+resource "ciphertrust_hsm_root_of_trust_setup" "cm_hsm_rot_setup" {
+  type         = "lunapci"
+  conn_info = {
+    partition_name     = "kylo-partition"
+    partition_password = "sOmeP@ssword"
+  }
+  reset = true
+  delay = 5
```
Comment (VJ-VijayThales) [inline docs/resources/hsm_root_of_trust_setup.md:67]: dont we need to change the delay here ?

File: examples/resources/ciphertrust_hsm_root_of_trust_setup/resource.tf [hunk 2]
```diff
@@ -1,9 +1,9 @@
 # Terraform Configuration for CipherTrust Provider
 
-# This configuration demonstrates the creation of an HSM Root of trust setup of type luna
+# These configurations demonstrates the creation of an HSM Root of trust setup of type "luna", "lunapci"
```
Comment (VJ-VijayThales) [inline examples/resources/ciphertrust_hsm_root_of_trust_setup/resource.tf:3]: as you make it plural, i think then everything has to be in plural sense
1.demonstrates
2.types
:)

File: examples/resources/ciphertrust_hsm_root_of_trust_setup/resource.tf [hunk 3]
```diff
@@ -36,4 +36,15 @@ resource "ciphertrust_hsm_root_of_trust_setup" "cm_hsm_rot_setup" {
   }
   reset = true
   delay = 5
```
Comment (VJ-VijayThales) [inline examples/resources/ciphertrust_hsm_root_of_trust_setup/resource.tf:38]: on yesterdays call, you said we have to provide more delay right.. or was this enough? im not sure.. can you clarify that

File: docs/resources/hsm_root_of_trust_setup.md (hunk 1 above)
Comment (om-shrirao) [reply docs/resources/hsm_root_of_trust_setup.md:67]: no it is fine.

File: examples/resources/ciphertrust_hsm_root_of_trust_setup/resource.tf (hunk 3 above)
Comment (om-shrirao) [reply examples/resources/ciphertrust_hsm_root_of_trust_setup/resource.tf:38]: no that was for unit tests.

**Pull request merged**